docs/development/hosting.md

```diff
@@ -46,11 +27,9 @@ The full publishing flow is as follows:
    does any internal checks.  These checks may include branch checks,
    verification that an authorized user made the commit, checks for passing
    tests, or other checks.
-0. Jenkins will set the `TESTPILOT_AMO_USER` and `TESTPILOT_AMO_SECRET`
-   environment variables
-0. Jenkins will run `bin/build-addon.sh`
-0. Jenkins will prepare appropriate S3 files (the signed .xpi file, /latest/
-   redirect, a new updates.json file)
+0. Jenkins will run `npm install` followed by `npm run package`
```

Comment:
Yeah, I'd rather the "API" here be a shell script than an npm script, just for flexibility's sake